src/components/pages/import-batch-view.js

```diff
@@ -1,24 +1,58 @@
-import React from 'react';
 import Page from '../page.js';
 import PropTypes from 'prop-types';
+import Logger from '../../common/logger.js';
 import { useTranslation } from 'react-i18next';
+import React, { useEffect, useState } from 'react';
+import { useService } from '../container-context.js';
 import { useDateFormat } from '../language-context.js';
+import { handleApiError } from '../../ui/error-helper.js';
 import { getImportedArticleUrl } from '../../utils/urls.js';
 import { DOCUMENT_IMPORT_TYPE } from '../../common/constants.js';
+import ImportApiClient from '../../services/import-api-client.js';
 import { Table, Row, Space, Tooltip, Collapse, List } from 'antd';
 import { importBatchDetailsShape } from '../../ui/default-prop-types.js';
-import { CloudDownloadOutlined, CloudSyncOutlined, WarningOutlined, CheckOutlined, ExclamationCircleOutlined, SyncOutlined } from '@ant-design/icons';
 import { isTaskSuccessful, taskStatusSorter, doesTaskHaveErrors } from '../../utils/task-utils.js';
+import { CloudDownloadOutlined, CloudSyncOutlined, WarningOutlined, CheckOutlined, ExclamationCircleOutlined, SyncOutlined } from '@ant-design/icons';
 
 const { Panel } = Collapse;
+const POLL_INTERVAL_IN_MS = 1000;
```

Comment:
Maybe 1s is a bit too often here 🤔 For a big payload especially.

Reply:
It felt alright, I had it with 2 seconds and it felt like an eternity :)). I can put it higher again. Suggest a value (try it locally with "all of stage" data :)).

Reply:
Then let's split the difference and make it 1.5s?

Reply:
Sure, done